tests/snapshot/snapshot/restore.yaml

```diff
@@ -19,6 +19,12 @@ epilogues:
   - path: /books
     method: DELETE
     status: [200, 404]
+  - path: /stories
+    method: DELETE
+    status: [200, 404]
+  - path: /new_stories
```

Comment:
We should call it something meaningful, maybe `tales`?

Reply:
Do you mean `new_stories`? This is the restored-with-new-name index. I changed it to `renamed_stories`.

Reply:
I just don't love names like `new_xyz` or `xyz_1` because we plan to use these samples to generate documentation. It's better with names like "movies" or "films", so instead of "new_stories" maybe we can use something semantically interesting like "tales"?

Reply:
This name is for the rename parameters for this API. An entirely new name like that would not fit the most common use case of those parameters, which is to restore a copy of existing index with a new, non-conflicting name.  So, even for documentation, I think a prefixed name like currently is more suitable. The [current documentation of restore](https://opensearch.org/docs/latest/api-reference/snapshots/restore-snapshot/) supports this - it has an example of a prefixed name, `recovered-logs`, and a suffixed name, `opendistro-reports-definitions_restored`, but no example of a entirely new name for renamed indexes.

Reply:
Understood, thanks.

Reply:
I have renamed this index and rewrote the rename parameters to match realistic use cases.